ManagementCenterServiceTest runScript

- Fails on `Hazelcast-4.maintenance-ZuluJDK15`
- Fails on [Build #3 (Nov 8, 2020 7:30:42 AM)](http://jenkins.hazelcast.com/view/Official%20Builds/job/Hazelcast-4.maintenance-ZuluJDK15/3/testReport/junit/com.hazelcast.client.management/ManagementCenterServiceTest/runScript/)
- Error
```
java.lang.IllegalArgumentException: Could not find ScriptEngine named 'javascript'.
```
- Stacktrace
```
Caused by: java.lang.IllegalArgumentException: Could not find ScriptEngine named 'javascript'.
	at com.hazelcast.internal.management.operation.RunScriptOperation.lambda$run$0(RunScriptOperation.java:65)
	at com.hazelcast.internal.util.executor.CompletableFutureTask.run(CompletableFutureTask.java:64)
	at com.hazelcast.internal.util.executor.CachedExecutorServiceDelegate$Worker.run(CachedExecutorServiceDelegate.java:217)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.lang.Thread.run(Thread.java:832)
	at com.hazelcast.internal.util.executor.HazelcastManagedThread.executeRun(HazelcastManagedThread.java:64)
	at com.hazelcast.internal.util.executor.HazelcastManagedThread.run(HazelcastManagedThread.java:80)
	at ------ submitted from ------.()
	at com.hazelcast.internal.util.ExceptionUtil.cloneExceptionWithFixedAsyncStackTrace(ExceptionUtil.java:265)
	at com.hazelcast.spi.impl.operationservice.impl.InvocationFuture.returnOrThrowWithGetConventions(InvocationFuture.java:112)
```
- kindly check

--- a/hazelcast/src/test/java/com/hazelcast/client/management/ManagementCenterServiceTest.java
+++ b/hazelcast/src/test/java/com/hazelcast/client/management/ManagementCenterServiceTest.java
@@ -30,6 +30,7 @@ import com.hazelcast.core.HazelcastInstance;
 import com.hazelcast.instance.BuildInfoProvider;
 import com.hazelcast.internal.cluster.impl.VersionMismatchException;
 import com.hazelcast.internal.json.JsonObject;
+import com.hazelcast.internal.management.ScriptEngineManagerContext;
 import com.hazelcast.internal.management.dto.ClientBwListDTO;
 import com.hazelcast.internal.management.dto.ClientBwListEntryDTO;
 import com.hazelcast.internal.management.dto.MCEventDTO;
@@ -46,6 +47,8 @@ import org.junit.Test;
 import org.junit.experimental.categories.Category;
 import org.junit.runner.RunWith;
 
+import javax.script.ScriptEngine;
+import javax.script.ScriptEngineManager;
 import java.security.AccessControlException;
 import java.util.List;
 import java.util.Map;
@@ -74,6 +77,7 @@ import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertNull;
 import static org.junit.Assert.assertThat;
 import static org.junit.Assert.assertTrue;
+import static org.junit.Assume.assumeNotNull;
 
 @RunWith(HazelcastParallelClassRunner.class)
 @Category({QuickTest.class, ParallelJVMTest.class})
@@ -316,7 +320,13 @@ public class ManagementCenterServiceTest extends HazelcastTestSupport {
 
     @Test
     public void runScript() throws Exception {
-        String result = resolve(managementCenterService.runScript(members[0], "javascript", "'hello world';"));
+        String engineName = "javascript";
+
+        ScriptEngineManager scriptEngineManager = ScriptEngineManagerContext.getScriptEngineManager();
+        ScriptEngine scriptEngine = scriptEngineManager.getEngineByName(engineName);
+        assumeNotNull(scriptEngine);
+
+        String result = resolve(managementCenterService.runScript(members[0], engineName, "'hello world';"));
         assertEquals("hello world", result);
     }
 

